docs/development.rst

```diff
@@ -1355,6 +1355,12 @@ Once all this is done, submit a PR containing the code changes and wait for dire
 on whom to send the compressed archive of genetic maps to (currently Andrew Kern is the
 primary uploader but please wait to send files to him until directed).
 
+An important note: when an existing resource file (such as a genetic map or annotation)
+is updated and replaces the previous version,
+be sure to give it a new version name (i.e. ``name_{id}_more_name_v1.txt``). 
+This is to ensure that the previous file versions and checksums that are hosted
+remotely remain valid when new releases are made.
+
```

Comment:
Is this sufficient and what you had in mind @petrelharp?

Reply:
Not quite. Can you add, let's see:
1. in what files do you need to put the new name
2. something more prescriptive about how to create the new name (your example is not following how you created the new names for this round - and, they are .tar.gz, not .txt, right?)
3. a note that in the cache the files are renamed to a standard thing
4. something saying why you have to do this?

Reply:
Updates! Better?

Reply:
Went ahead with updating names to match what I described in docs.

Reply:
Sounds good - that made more sense to me, but I assumed you had a good reason for doing it the other way.